<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginsPage</class>
 <widget class="QWidget" name="PluginsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>12</number>
     </property>
     <item>
      <widget class="QLabel" name="labelPlugins">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Plugins</string>
       </property>
       <property name="indent">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bImport">
       <property name="styleSheet">
        <string notr="true">QPushButton  {
  border: 1px solid white;
  border-radius: 8px;
  background-color: transparent;
  padding-left: 10px;
  padding-right: 10px;
}
QPushButton::hover {
  background-color: #4285f4;
}</string>
       </property>
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="iwLayout">
     <item>
      <widget class="QTableWidget" name="table">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>type</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>uid</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QDockWidget" name="dockBrowser">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="features">
        <set>QDockWidget::NoDockWidgetFeatures</set>
       </property>
       <property name="allowedAreas">
        <set>Qt::RightDockWidgetArea</set>
       </property>
       <widget class="QWidget" name="dockWidgetContents_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTextBrowser" name="browser">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(24, 24, 24);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
